src/main/java/it/pagopa/transactions/exceptions/TransactionAmountMismatchException.java

```diff
@@ -0,0 +1,15 @@
+package it.pagopa.transactions.exceptions;
+
+public class TransactionAmountMismatchException extends RuntimeException {
+
+    public TransactionAmountMismatchException(
+            Integer requestAmount,
+            Integer transactionAmount
+    ) {
+        super(
+                "Transaction amount mismatch! Request amount: [%s], Transaction amount: [%s]"
```

Comment:
focus point, is it correct to return this message to the client? it is important to log (with a dedicated exception) but the client doesn't have to know the correct amount in the storage

Reply:
You're right. Modified into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/117/commits/8c15e62e172a81ad3ac5e21333338166bae130a6